README.md

```diff
@@ -133,3 +133,60 @@ var intValue: Int
 var intValue : Int
 
 ```
+
+### Enums ###
+
+For enum declarations, declare each enum case on a new line with its own `case` statement instead of a comma-separated list.
+
+```swift
+internal enum State
```

Comment:
I think it's overkill to specify since we all know that internal is the default behavior, what do you think?